style.css

```diff
@@ -0,0 +1,138 @@
+.header {
+    background-color: rgba(11, 12, 12, 0.89);
+    width: 378px;
+    height: 92px;
+    position: absolute;
+  
+  }
+  body{
+  margin: 0%;
+  padding: 0%;
+  width: 378px;
+   }
+  /* contect info start */
+  .contect-info { 
+      width: 100%;
+      height: 7vh;
```

Comment:
It would be better if you review again how to make a website more responsive it will be beneficial to you.